ui/src/logs/utils/config.ts

```diff
@@ -48,7 +54,7 @@ export const columnServerToUI = (column: ServerColumn): LogsTableColumn => {
   const encodings: LogsTableColumn = column.encodings.reduce(
     (acc, e) => {
       if (e.type === EncodingTypes.visibility) {
-        if (e.value === 'visible') {
+        if (e.value === EncodingVisibilityOptions.visible) {
```

Comment:
Could we use `&&` instead of the nested if?

Reply:
👍